RemoteHealthcare/HRManager.cs

```diff
@@ -14,11 +11,7 @@ class HRManager
         private static int OriginalrequestedDataAmount = 0;
         private static bool exit = false;
         private static bool reachedThreshold = false;
-
         private static BLE bleHeart = null;
-        public HRManager()
-        {
-        }
 
         public async void MakeConnection(int dataBlocks)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)